agent/src/tools/search_exa_people.py

```
@@ -0,0 +1,141 @@
+                timeout=20.0,
+            )
+        response.raise_for_status()
+        data = response.json()
+    except httpx.HTTPStatusError as exc:
+        return {
+            "status": "error",
+            "error": f"Exa API returned {exc.response.status_code}",
+            "error_category": "search_tool_error",
+            "source": "exa",
+        }
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)